R/match_on_ltcs.R

```diff
@@ -0,0 +1,100 @@
+#' Match on LTC changed_dob and dates of LTC incidence (based on hospital incidence only)
+#'
+#' @description Match on LTC changed_dob and dates of LTC incidence (based on hospital incidence only)
+#'
+#' @param
+#'
+#' @return
+#' @export
+#'
+#' @examples
+match_on_ltcs <- function(data, year) {
```

Comment:
These will need properly filling out
```suggestion
#' @param data ...
#' @param year ...
#'
#' @return ...
#' @export
match_on_ltcs <- function(data, year) {
```